MapboxCoreNavigationTests/DistanceFormatterTests.swift

```diff
@@ -87,15 +85,15 @@ class DistanceFormatterTests: XCTestCase {
         assertDistance(999,     displayed: "1 km",      quantity: "1")
         assertDistance(1000,    displayed: "1 km",      quantity: "1")
         assertDistance(1001,    displayed: "1 km",      quantity: "1")
-        assertDistance(2_500,   displayed: "2.5 km",    quantity: "2.5")
-        assertDistance(2_900,   displayed: "2.9 km",    quantity: "2.9")
+        assertDistance(2_500,   displayed: "2,5 km",    quantity: "2,5")
+        assertDistance(2_900,   displayed: "2,9 km",    quantity: "2,9")
```

Comment:
I don’t think this indicates a bug in master, just a shortcoming of how the test was previously written. The test previously set the locale of the number formatter above, but the locale would be changed back to `nationalizedCurrent` within the formatter. Now the locale is set on the distance formatter, which takes care of the measurement formatter’s locale.